src/main/java/com/lowagie/text/rtf/RtfMapper.java

```diff
@@ -119,14 +119,14 @@ public RtfMapper(RtfDocument doc) {
      * @throws DocumentException
      */
     public RtfBasicElement[] mapElement(Element element) throws DocumentException {
-        ArrayList rtfElements = new ArrayList();
-
-        if(element instanceof RtfBasicElement) {
+		if(element instanceof RtfBasicElement) {
             RtfBasicElement rtfElement = (RtfBasicElement) element;
             rtfElement.setRtfDocument(this.rtfDoc);
             return new RtfBasicElement[]{rtfElement};
         }
-        switch(element.type()) {
+
+		ArrayList<RtfBasicElement> rtfElements = new ArrayList<>();
```

Comment:
For future PR: It is a good practice to declare the variable using the interface, and not the implementation, so this line could be written as.
`List<RtfBasicElement> rtfElements = new ArrayList<>();`
This can also be applied to method-Parameters. Care should be taken, when modifying  the return type of a method. as this could break the code of the users.

Reply:
I left the code like this unchanged. It is a good idea to use collection interfaces, but it is complicated by the fact that there is commonly used com.lowagie.text.List type.

Is the goal to keep binary or source compatibility?

Reply:
Binary. So changing could be a pain, if it is the return type.